ci/verify-chefdk.sh

```diff
@@ -2,6 +2,9 @@
 
 set -evx
 
+# Cleanning up some cruft from previous tests
+sudo find /tmp -name 'chef-dk*' | sudo xargs rm -rf
```

Comment:
yeah I considered that too but then noticed the `sudo` changes were only for the acceptance tests. I am gonna look at the testers again and see if these chef-dk* directories are still being created. For all I know these directories aren't even created anymore and this entire PR is unnecessary.